### PR TITLE
Fix more modules config files

### PR DESCRIPTION
**Pull request by iliajie**
Fix more modules config files

### CODE REVIEW

File: apache/config-AlmaLinux-7.0-ALL [hunk 1]
```diff
@@ -1,4 +1,4 @@
-show_list=0
+show_list=1
```
Comment (jcameron) [inline apache/config-AlmaLinux-7.0-ALL:1]: Why make this change?

**Comment by iliajie** [comment]
Sorry for not putting this to a separate commit but it's just cosmetics. The point is to have Apache display hosts as a table, which you can type and search for a domain name. Also there is a button to open domain. I find it much more useful then icons, especially if there a lot of domains.

Do you mind changing it?

**Comment by iliajie** [comment]
I have taken `show_list` out from this PR. Tell me if you're okay to switching to table layout. I will do it later.

**Comment by iliajie** [comment]
I just realized I need to add CentOS Stream support for configs too!

File: apache/config-debian-linux-7.0-9.0 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-show_list=0
+show_list=1
```
Comment (jcameron) [inline apache/config-debian-linux-7.0-9.0:1]: Why change show_list in this PR?

Comment (iliajie) [reply apache/config-debian-linux-7.0-9.0:1]: A mistake.. Let me clean it.

Comment (iliajie) [reply apache/config-debian-linux-7.0-9.0]: Fixed.

**Pull request merged**